### PR TITLE
Kirti/language modal

### PR DESCRIPTION
**Pull request by kirti18patel**
Fix header height
Add language Modal

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -71,3 +71,7 @@ a {
 	transform: translateX(-50%);
 	width: var(--swiper-scrollbar-width) !important;
 }
+
+.shadow-custom {
```
Comment (Shaderpixel) [inline styles/globals.css:75]: this shouldn't be added here, it should be using CSS modules, please reference 9thCO implementation look for files that ends with [name].module.css

**Review by Shaderpixel — CHANGES_REQUESTED** [review]
Some minor changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,9 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.4] -05/17/2023
+
+### Added
+
+-   Add language modal
+
+### Changed
```
[the remark this answers is not in the document]
Comment (kirti18patel) [reply CHANGELOG.md:14]: removed unused Changed and Removed headers

**Review by Shaderpixel — APPROVED** [review]
looks good.

**Pull request merged**